nodejs/builder/build.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 cd ${SRC_PKG}
+if [[ -n "NPM_TOKEN" ]] && [[ -n "NPM_REGISTRY" ]]; then
```

Comment:
This should be ?
```
if [[ -n "$NPM_TOKEN" ]] && [[ -n "$NPM_REGISTRY" ]]; then
```
It would evaluate to true always otherwise, I guess.

Reply:
thanks, fixed